sdcm/cluster.py

```diff
@@ -4171,6 +4179,11 @@ def reconfigure_scylla_monitoring(self):
                     """.format(self))
             node.remoter.run("sudo bash -ce '%s'" % configure_script, verbose=True)
 
+            configure_self_node_exporter = dedent(f'''
+                docker run --rm -v {self.monitoring_conf_dir}:/workdir mikefarah/yq yq w -i node_exporter_servers.yml '[0].targets[+]' ''[{node.private_ip_address}]:9100''
+            ''')
+            node.remoter.run("sudo bash -ce '%s'" % configure_self_node_exporter, verbose=True)
+
```

Comment:
Few lines below we use much simpler way to alter same config. Do we really need to fire from the Docker cannon to kill this a pretty small bird?

Reply:
In the section below, we are replacing the whole file, not altering it, the other option it to copy it back an forth, and edit localy

Reply:
Ok, got it.